.github/workflows/tests.yml

```diff
@@ -228,17 +228,16 @@ jobs:
 
       matrix:
         node-version:
-          - 12.18 # Node.js 12.18 uses package exports with trailing slashes
-          - 12 # But Node.js 12.20+ uses package exports with subpath patterns
-          - 18
+          - 12 # Node.js 12.20+ uses package exports with subpath patterns
+          - 18 # Node.js 17+ cannot use package exports with trailing slashes
 
         conditions:
-          - require
-          - import
+          - [require]
```

Comment:
Yeah fair shout, we can save this for another day

For context, we spiked **package.json** `"browser"` exports pre-bundled but otherwise separate modules:

```json
".": {
  "browser": {
    "sass": "./dist/govuk/all.scss",
    "import": "./dist/govuk/all.bundle.mjs",
    "require": "./dist/govuk/all.bundle.js",
    "default": "./dist/govuk/all.bundle.js"
  },
  "sass": "./dist/govuk/all.scss",
  "import": "./dist/govuk/all.mjs",
  "require": "./dist/govuk/all.cjs",
  "default": "./dist/govuk/all.bundle.js"
}
```

Would have needed those CommonJS exports though so we dropped it

```suggestion
          - [browser, require]
```

Reply:
Split out into separate spike PR:

* https://github.com/alphagov/govuk-frontend/pull/3920